services/web-app/next.config.js

```diff
@@ -55,6 +55,11 @@ module.exports = {
       })
     })
 
+    config.experiments = {
+      ...config.experiments,
+      topLevelAwait: true
+    }
+
```

Comment:
I know we'd mentioned wanting to get rid of the top-level awaits, if possible. We can talk through options for resolving that.

Reply:
wrapped all top-level awaits in async handler